docassemble/MOHUDEvictionProject/data/questions/MOHUDEvictionProject.yml

```diff
@@ -521,6 +504,51 @@ subquestion: |
   A counterclaim is a reason why your landlord should pay you money.
 continue button field: signpost_claims_and_defenses
 ---
+id: do you agree with the landlord
+question: |
+  Landlord's reason for eviction
+subquestion: |
+  % if person_answering == "tenant":
+  Your landlord says that they are evicting you because:
+  % else:
+  The landlord says that they are evicting the tenant because:
+  % endif
+
+  % for reason in eviction_reason.true_values():
+   - ${ reason }
+  % endfor
+
+  % if person_answering == "tenant":
+  Do you agree with your landlord's reason for eviction, or do you deny it?
+
+  If at least **one** of the reasons or facts that the landlord
+  claims about you is not true, you should deny the landlord's reason for eviction.
+  % else:
+  Does the tenant agree with the landlord's reason for eviction?
+
+  If at least **one** of the reasons or facts that the landlord
+  claims about the tenant is not true, you should deny the landlord's reason for eviction.
+  % endif  
+fields:
+  - label: |
+      % if person_answering == "tenant":
+      I deny doing what the landlord says that I did
+      % else:
+      The tenant denies doing what the landlord says that they did
+      % endif
+    field: defense_no_breach
+    datatype: yesnoradio
+  - label: |
+      % if person_answering == "tenant":
+      Even if what the landlord says happened is true, I deny that the reason the landlord gave is a good enough reason
+      to evict me.
+      % else:
+      Even if what the landlord says happened is true, the tenant denies that the reason the landlord gave is a good enough reason
+      to evict the tenant.
+      % endif
+    field: defense_breach_not_material
+    datatype: yesnoradio
+---
 id: defenses explanation
```

Comment:
The first portion of this question appears to only be for rent and passion cases, but showed on UD path. Does it apply to both? Might just be a language change?

**Landlord failed to state a claim for rent and possession**
  A landlord must include specific things in the landlord's petition in order to be able to evict.  Your landlord failed to include the following things: 

![Screen Shot 2023-09-22 at 8 29 23 AM](https://github.com/LemmaLegalConsulting/docassemble-MOHUDEvictionProject/assets/65858862/c1228992-9ae1-4392-8c87-796e968207de)